lib/tokenisers.py

```diff
@@ -11,11 +11,13 @@ def quc(sentence):
     return [i for i in re.split("(\\w+)", o) if not i.strip() == ""]
 
 
-def azz(sentence):
+def _azz(sentence):
     """Tokeniser for azz.
 
-    >>> tokenise('mayejyeualtik tepitsin saj. *Bueno** amo mapijpitsakueyak, *sino que** mamej..., mayejyeualtik uaukilit.', lang='azz')
-    ['mayejyeualtik', 'tepitsin', 'saj', '.', '*', 'Bueno', '**', 'amo', 'mapijpitsakueyak', ',', '*', 'sino', 'que', '**', 'mamej', '...,', 'mayejyeualtik', 'uaukilit', '.']
+        >>> tokenise('mayejyeualtik tepitsin saj. *Bueno** amo mapijpitsakueyak, *sino que** mamej...,
```

Comment:
We could make this example much simpler with keyword arguments and black style formatting (one argument per line).  I thought I had blackdocs turned on but maybe the line length needs to be tweaked for that hook.  I'll take a peek at this one later so ignore in this review.

Reply:
I don't want to make the example shorter as it has some useful stuff in there. How would keyword arguments make it simpler?

Reply:
It'd make it more clear.  I don't think the content should change, but the keyword arguments make it clear what is intended at each point.  The black formatting would put this into a multiline format which should get closer to the line limit.  Not sure on all of these though.